docs/LANGUAGE_TRANSLATION.md

```diff
@@ -145,7 +131,8 @@ constructor() {
    orm,
    oji,
    dak,
-   vie
+   vie,
+   ru
```

Comment:
I think we've mostly been using three-letter codes for languages, could change this to `rus` or another three-letter code?